deepcell/utils/transform_utils_test.py

```diff
@@ -105,34 +105,40 @@ def test_pixelwise_transform_3d(self):
             batch_count = maskstack.shape[0] // frames
```

Comment:
I wasn't sure why the pixel_transform test went with this approach, compared to the watershed_transform test, so I just looped through the stack for now

Reply:
I also do not know why this approach was used, I think what you have done is fine.